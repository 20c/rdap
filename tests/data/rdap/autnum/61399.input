{
  "handle": "AS61399",
  "startAutnum": 61399,
  "endAutnum": 61399,
  "name": "KG-IX",
  "status": [
    "active"
  ],
  "entities": [
    {
      "handle": "kg-ix-1-mnt",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "kg-ix-1-mnt"
          ],
          [
            "kind",
            {},
            "text",
            "individual"
          ]
        ]
      ],
      "roles": [
        "registrant"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/61399",
          "rel": "self",
          "href": "https://rdap.db.ripe.net/entity/kg-ix-1-mnt"
        },
        {
          "value": "http://www.ripe.net/data-tools/support/documentation/terms",
          "rel": "copyright",
          "href": "http://www.ripe.net/data-tools/support/documentation/terms"
        }
      ],
      "objectClassName": "entity"
    },
    {
      "handle": "MP31159-RIPE",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "Madina Botirova"
          ],
          [
            "kind",
            {},
            "text",
            "individual"
          ],
          [
            "adr",
            {
              "label": "Razzakova 55\n720000\nBishkek\nKYRGYZSTAN"
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "tel",
            {
              "type": "voice"
            },
            "text",
            "+996555924135"
          ]
        ]
      ],
      "roles": [
        "technical",
        "administrative"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/61399",
          "rel": "self",
          "href": "https://rdap.db.ripe.net/entity/MP31159-RIPE"
        },
        {
          "value": "http://www.ripe.net/data-tools/support/documentation/terms",
          "rel": "copyright",
          "href": "http://www.ripe.net/data-tools/support/documentation/terms"
        }
      ],
      "objectClassName": "entity"
    },
    {
      "handle": "ORG-KL219-RIPE",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "KG-IX LLC"
          ],
          [
            "kind",
            {},
            "text",
            "org"
          ],
          [
            "adr",
            {
              "label": "Razzakova st. 55\n720040\nBishkek\nKYRGYZSTAN"
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "tel",
            {
              "type": "voice"
            },
            "text",
            "+996555924135"
          ],
          [
            "email",
            {
              "type": "email"
            },
            "text",
            "lir@ix.kg"
          ]
        ]
      ],
      "roles": [
        "registrant"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/61399",
          "rel": "self",
          "href": "https://rdap.db.ripe.net/entity/ORG-KL219-RIPE"
        },
        {
          "value": "http://www.ripe.net/data-tools/support/documentation/terms",
          "rel": "copyright",
          "href": "http://www.ripe.net/data-tools/support/documentation/terms"
        }
      ],
      "objectClassName": "entity"
    },
    {
      "handle": "RIPE-NCC-END-MNT",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "RIPE-NCC-END-MNT"
          ],
          [
            "kind",
            {},
            "text",
            "individual"
          ],
          [
            "org",
            {},
            "text",
            "ORG-NCC1-RIPE"
          ]
        ]
      ],
      "roles": [
        "registrant"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/61399",
          "rel": "self",
          "href": "https://rdap.db.ripe.net/entity/RIPE-NCC-END-MNT"
        },
        {
          "value": "http://www.ripe.net/data-tools/support/documentation/terms",
          "rel": "copyright",
          "href": "http://www.ripe.net/data-tools/support/documentation/terms"
        }
      ],
      "objectClassName": "entity"
    },
    {
      "handle": "AR41993-RIPE",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "Abuse-C Role"
          ],
          [
            "kind",
            {},
            "text",
            "group"
          ],
          [
            "adr",
            {
              "label": "271 Frunze st.\n720065\nBishkek\nKYRGYZSTAN"
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "email",
            {
              "type": "email"
            },
            "text",
            "lir@ix.kg"
          ],
          [
            "email",
            {
              "type": "abuse"
            },
            "text",
            "lir@ix.kg"
          ]
        ]
      ],
      "roles": [
        "abuse"
      ],
      "entities": [
        {
          "handle": "kg-ix-1-mnt",
          "vcardArray": [
            "vcard",
            [
              [
                "version",
                {},
                "text",
                "4.0"
              ],
              [
                "fn",
                {},
                "text",
                "kg-ix-1-mnt"
              ],
              [
                "kind",
                {},
                "text",
                "individual"
              ]
            ]
          ],
          "roles": [
            "registrant"
          ],
          "links": [
            {
              "value": "https://rdap.db.ripe.net/autnum/61399",
              "rel": "self",
              "href": "https://rdap.db.ripe.net/entity/kg-ix-1-mnt"
            },
            {
              "value": "http://www.ripe.net/data-tools/support/documentation/terms",
              "rel": "copyright",
              "href": "http://www.ripe.net/data-tools/support/documentation/terms"
            }
          ],
          "objectClassName": "entity"
        }
      ],
      "objectClassName": "entity"
    }
  ],
  "links": [
    {
      "value": "https://rdap.db.ripe.net/autnum/61399",
      "rel": "self",
      "href": "https://rdap.db.ripe.net/autnum/61399"
    },
    {
      "value": "http://www.ripe.net/data-tools/support/documentation/terms",
      "rel": "copyright",
      "href": "http://www.ripe.net/data-tools/support/documentation/terms"
    }
  ],
  "events": [
    {
      "eventAction": "registration",
      "eventDate": "2017-07-10T09:15:46Z"
    },
    {
      "eventAction": "last changed",
      "eventDate": "2022-09-17T13:16:18Z"
    }
  ],
  "rdapConformance": [
    "nro_rdap_profile_asn_flat_0",
    "cidr0",
    "rdap_level_0",
    "nro_rdap_profile_0"
  ],
  "notices": [
    {
      "title": "Filtered",
      "description": [
        "This output has been filtered."
      ]
    },
    {
      "title": "Whois Inaccuracy Reporting",
      "description": [
        "If you see inaccuracies in the results, please visit:"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/61399",
          "rel": "inaccuracy-report",
          "href": "https://www.ripe.net/contact-form?topic=ripe_dbm&show_form=true",
          "type": "text/html"
        }
      ]
    },
    {
      "title": "Source",
      "description": [
        "Objects returned came from source",
        "RIPE"
      ]
    },
    {
      "title": "Terms and Conditions",
      "description": [
        "This is the RIPE Database query service. The objects are in RDAP format."
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/61399",
          "rel": "terms-of-service",
          "href": "http://www.ripe.net/db/support/db-terms-conditions.pdf",
          "type": "application/pdf"
        }
      ]
    }
  ],
  "port43": "whois.ripe.net",
  "objectClassName": "autnum"
}