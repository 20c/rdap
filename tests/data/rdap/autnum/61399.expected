{
  "name": "KG-IX",
  "emails": [
    "lir@ix.kg"
  ],
  "org_name": "KG-IX LLC",
  "org_address": "Razzakova st. 55\n720040\nBishkek\nKYRGYZSTAN"
}